tests/sentry/api/endpoints/test_project_group_index.py

```diff
@@ -362,6 +362,39 @@ def test_global_resolve(self):
             group=new_group4,
         )
 
+    def test_bulk_resolve(self):
+        self.login_as(user=self.user)
+
+        for i in range(200):
+            self.create_group(checksum=six.binary_type(i) * 2, status=GroupStatus.UNRESOLVED)
```

Comment:
I don't think you actually need to pass `checksum` at all: https://github.com/getsentry/sentry/blob/6e47bc607827da5fb8a9976e31d86d59c3607465/src/sentry/testutils/fixtures.py#L523-L525